projecttacoma/deqm-test-server-frontend
# back buttons rendered, unit tested, ready for review

## Summary
Back buttons on the create page and [id] page
## New behavior
Back buttons are rendered and functional on the create page and [id] page. Clicking on a back button takes user to the page they were one previously. Clicking on app header loads home page.
## Code changes
Main changes in: 
- pages/[resourceType]/[id].tsx 
- pages/[resourceType]/create.tsx

Unit tests added to: 
- tests/pages/[resource]/create.test.tsx 
- tests/pages/[resource]/[id].test.tsx

Minor UI changes in: 
- components/ResourceIDs.tsx pages/_app.tsx 
- pages/[resourceType]/index.tsx

## Testing guidance
See README.md for first time setup. Start the frontend by running`npm run dev`
- Test back buttons in browser:
  1. Navigate to http://localhost:3000 in a browser.
  2.  Then click on any resource in the navigation menu
  3.   Click on any of the resource IDs **OR** click on the Create New [resourcyType] button 
  4. Then click on the Back button in the upper left hand corner of the main page body. This should lead to back to the page you were last on.
 
- Click on the DEQM Test Server Frontend header, this will take you back to the home (empty main page body)

- Run the unit tests:
`npm run test`
